src/dependencies.jl

```diff
@@ -998,7 +998,7 @@ macro install(_libmaps...)
                         depsfile_buffer = IOBuffer()
                         println(depsfile_buffer,
                             """
-                            # This is an auto-generated file; do not edit
+                        # This is an auto-generated file; do not edit and do not check-in in version control
```

Comment:
Is there a reason you un-indented this?  Also, I believe the proper preposition is "do not check-in to version control".